alphadia/workflow/base.py

```diff
@@ -102,7 +102,9 @@ def load(
             self._calibration_manager.disable_mobility_calibration()
 
         # initialize the optimization manager
-
+        self.config["optimization_manager"]["gradient_length"] = (
+            self.dia_data.rt_values.max()
```

Comment:
I think we can avoid to edit the config dynamically.

Maybe we can just use workflow.dia_data.rt_values within the respective optimizer.

Reply:
I did it this way because the config is already dynamic (in the sense that the output folder field is only created and assigned during the planning phase), and I thought that perhaps we might want to be able to specify the gradient length in the config. But I am very happy to change it.

Reply:
I would also vote for the config being static (we could even enforce this using `MappingProxyType`: https://stackoverflow.com/a/72371333)